platform/viewer/cypress/support/commands.js

```diff
@@ -118,13 +130,24 @@ Cypress.Commands.add('waitDicomImage', (timeout = 10000) => {
 });
 
 
-//Command to reset the viewport changes throught the cornerstone method
+//Command to reset and clear all the changes made to the viewport
 Cypress.Commands.add('resetViewport', () => {
-  cy.get('@resetBtn').click()
+  cy.initCornerstoneToolsAliases();
+  cy.get('@resetBtn').click();
+  //Click on More button
+  cy.get('@moreBtn')
+    .click();
+  //Verify if overlay is displayed
+  cy.get('.tooltip-toolbar-overlay')
+    .should('be.visible');  
+  //Click on Clear button
+  cy.get('.tooltip-inner > :nth-child(10)')
+    .click();
```

Comment:
Yes, but only on this function scope because by the moment we initialize all aliases, this element is not visible. So a replacement would be:
```
cy.get('.tooltip-inner > :nth-child(10)').as('clearBtn')
    .click();
```